pyaerocom/dynamic_rec_array.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+
+
+class DynamicRecArrayException(Exception):
+    pass
+
+
+class DynamicRecArray:
+    def __init__(self, dtype, capacity: int = 10):
+        """A dynamic record based array of type dtype.
+
+        :param dtype: Datatype of the array. Must be readable by :class:`~numpy.dtype`
+        :param capacity: optional initial capacity
```

Comment:
What "capacity" really means I think would benefit from some explanation in very simple terms with an example in the doc string.